heron/schedulers/src/java/com/twitter/heron/scheduler/local/LocalLauncher.java

```diff
@@ -114,13 +120,36 @@ protected boolean setupWorkingDirectory() {
     String topologyPackageDestination = Paths.get(
         topologyWorkingDirectory, "topology.tar.gz").toString();
 
-    return SchedulerUtils.setupWorkingDirectory(
-        topologyWorkingDirectory,
-        coreReleasePackageURI,
-        coreReleaseFileDestination,
-        topologyPackageURI,
-        topologyPackageDestination,
-        Context.verbose(config));
+    if (!SchedulerUtils.setupWorkingDirectory(topologyWorkingDirectory)) {
```

Comment:
code on line 123 to line 148 is doing the actual setup work. Can we move them into the `SchedulerUtils. setupWorkingDirectory`? 

If there's a reason we couldn't put them there, we can create a local method here to make the code here clear.

Reply:
I wanted to separate the create/clean directory and the tar.gz file extraction. This decouples things for when a scheduler would like to use an installed version of the code. The extraction of the core.tar.gz is not needed but you had to pass those values in previous. Thoughts?